### PR TITLE
Standardized Outputs and Parameters 

### PR DESCRIPTION
**Pull request by EricFillion**
I believe it is best to stick to returning dictionaries for public methods that return multiple values to avoid confusion. We can discuss using a custom output class or using named tuples (as @ted537) suggested for the next major release.

I also noticed that the option parameter for predict_masks was named "masks_options" instead of "options" as per the predict_mask method. 

Changes: 
1. Changed predict_masks output from a list of lists that contain named tuples to a list of lists that contain dictionaries (similar to predict_mask).
2. Changed the answers_to_questions output from a list of named tuples to a list of dictionaries
3. Changed predict_masks "masks_options" parameter to "options" (to match predict_mask)
4. Added predict_masks() section to README
5. Updated readme documentation for answers_to_questions
6. Updated test cases answers_to_questions
7. Updated test cases for predict_masks

### CODE REVIEW

**Comment by ted537** [comment]
The reason for naming `masks_options` was to distinguish between the options parameters for `predict_mask()` and `predict_masks()`. `masks_options` is a 2D list, whereas `mask_options` is a 1D list. If we are unifying the naming, then it may be worth considering adding a friendly error message for the case where a 1D list is (wrongly) supplied as the options argument to `predict_masks()`.

TLDR: make sure people don't get tripped up on this

```
happy_bert = HappyBERT()
happy_bert.predict_masks('[MASK] is better than [MASK]',['hot','cold']) # we want a 2d list, these options don't make sense
```

**Review by ted537 — APPROVED** [review]

**Comment by EricFillion** [comment]
> The reason for naming `masks_options` was to distinguish between the options parameters for `predict_mask()` and `predict_masks()`. `masks_options` is a 2D list, whereas `mask_options` is a 1D list. If we are unifying the naming, then it may be worth considering adding a friendly error message for the case where a 1D list is (wrongly) supplied as the options argument to `predict_masks()`.
> 
> TLDR: make sure people don't get tripped up on this
> 
> ```
> happy_bert = HappyBERT()
> happy_bert.predict_masks('[MASK] is better than [MASK]',['hot','cold']) # we want a 2d list, these options don't make sense
> ```

That makes sense. However, I think staying consistent is key. For example, we use the parameter name "options" for both predict_mask and predict_masks.  For not we'll keep it as is and can consider adding an error message for if the user submits a list of strings.
